domains/_peerlist.mraxays.json

```
@@ -0,0 +1,6 @@
+{
+  "owner": { "username": "mraxays", "discord": "mrakshayas" },
```

fixing for readability
```suggestion
  "owner": {
      "username": "mraxays",
      "discord": "mrakshayas"
  },
```